cmd/containerd/main.go

```diff
@@ -119,11 +118,6 @@ high performance container runtime
 			runtime  = context.GlobalString("runtime")
 		)
 		switch runtime {
-		case "runc":
-			executor, err = oci.New(context.GlobalString("root"))
-			if err != nil {
-				return err
-			}
 		case "shim":
```

Comment:
Also, the default case error message is prefixed with "oci: "? Is that still desired?